### ISSUE TITLE
[mlir][standalone] Can't import linalg after building

### ISSUE THREAD

**Issue by qzylalala** [llvm/llvm-project#71473]
Hi, I build standalone successfully, but get an error when I try to use Python APIs.
```python3
from mlir_standalone.dialects import linalg
```
When I ran this line of code, I got an error like this
```
build/python_packages/standalone/mlir_standalone/dialects/linalg/__init__.py", line 6, in <module>
    from ..._mlir_libs._mlirDialectsLinalg import *
ImportError: build/python_packages/standalone/mlir_standalone/_mlir_libs/_mlirDialectsLinalg.cpython-38-x86_64-linux-gnu.so: undefined symbol: mlirLinalgFillBuiltinNamedOpRegion
```
I find that `mlirLinalgFillBuiltinNamedOpRegion` appears three times in llvm/mlir

1. `llvm/mlir/include/mlir-c/Dialect/Linalg.h`
https://github.com/llvm/llvm-project/blob/bdb309c5fd4030ae6ff9d5114e3532d45a98a183/mlir/include/mlir-c/Dialect/Linalg.h#L20-L25
2. `llvm/mlir/lib/Bindings/Python/DialectLinalg.cpp`
https://github.com/llvm/llvm-project/blob/bdb309c5fd4030ae6ff9d5114e3532d45a98a183/mlir/lib/Bindings/Python/DialectLinalg.cpp#L15-L28
3. `llvm/mlir/lib/CAPI/Dialect/Linalg.cpp`
https://github.com/llvm/llvm-project/blob/bdb309c5fd4030ae6ff9d5114e3532d45a98a183/mlir/lib/CAPI/Dialect/Linalg.cpp#L16-L42

However, I'm not sure what's the problem actually.

**Comment by xurui1995** [comment]
@qzylalala hi, Have you solved this problem？

**Comment by qzylalala** [comment]
Yes, but it's been so long that I don't remember the details, i'm sorry. Maybe you can reference [TPU-MLIR](https://github.com/sophgo/tpu-mlir)

**Comment by xurui1995** [comment]
> Yes, but it's been so long that I don't remember the details, i'm sorry. Maybe you can reference [TPU-MLIR](https://github.com/sophgo/tpu-mlir)

hi, thanks, i did not find any `import linalg` in the tpu mlir or some PRs about fixing this, When you have a moment, could you please provide me with more detailed information regarding this?

**Comment by qzylalala** [comment]
> > Yes, but it's been so long that I don't remember the details, i'm sorry. Maybe you can reference [TPU-MLIR](https://github.com/sophgo/tpu-mlir)
> 
> hi, thanks, i did not find any `import linalg` in the tpu mlir or some PRs about fixing this, When you have a moment, could you please provide me with more detailed information regarding this?

Maybe we can communicate via wechat: lemon1you

**Comment by xurui1995** [comment]
> > > Yes, but it's been so long that I don't remember the details, i'm sorry. Maybe you can reference [TPU-MLIR](https://github.com/sophgo/tpu-mlir)
> > 
> > 
> > hi, thanks, i did not find any `import linalg` in the tpu mlir or some PRs about fixing this, When you have a moment, could you please provide me with more detailed information regarding this?
> 
> Maybe we can communicate via wechat: lemon1you

Thank you. After including `MLIRPythonSources.Dialects.linalg` in my CAPI library sources, I am now able to import linalg successfully.
